nuxeo-distribution/nuxeo-nxr-server/src/main/resources/templates/common-base/nxserver/config/kafka-config.xml.nxftl

```diff
@@ -74,21 +80,22 @@
         <property name="max.poll.interval.ms">${kafka.max.poll.interval.ms}</property>
         <property name="session.timeout.ms">${kafka.session.timeout.ms}</property>
         <property name="heartbeat.interval.ms">${kafka.heartbeat.interval.ms}</property>
-<#if "${kafka.ssl}" == "true">
-<#if "${kafka.sasl.enabled}" == "true">
+<#if "${kafka.sasl.enabled}" == "true" || "${kafka.ssl}" == "true">
         <property name="security.protocol">${kafka.security.protocol}</property>
+</#if>
+<#if "${kafka.sasl.enabled}" == "true">
         <property name="sasl.mechanism">${kafka.sasl.mechanism}</property>
         <property name="sasl.jaas.config">${kafka.sasl.jaas.config}</property>
-<#else>
-        <property name="security.protocol">SSL</property>
 </#if>
-        <property name="max.poll.records">${kafka.max.poll.records}</property>
```

Comment:
If you really want to remove this and it's not an editing error, please do it in a separate ticket.

Reply:
Basically I think the bulk and non-bulk consumers and producers should have identical config. If not (check with @bdelbosc) the there should be a comment in the file explaining the differences, for future readers.

Reply:
This is an editing error, the diff should be only an additional #IF to check if there is an ssl keystore.

Reply:
Ok so @NourNuxeo  let's just move this line (which is inconsistently located in bulk and non-bulk) before all the ifs.

Reply:
It happens this line is a duplicate. It is already present on the top
```
<#if "${kafka.enabled}" == "true">
  ...
  <property name="max.poll.records">${kafka.max.poll.records}</property>
```

Reply:
@NourNuxeo you're mistaken, it's present in the `<kafkaConfig name="default"` section but it must also be present in the `<kafkaConfig name="bulk"` section.